labs/b4.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Beginner Lab 4 - Debian, packages, compiling software
+layout: lab
+---
+
+## What is a distribution?
+
+A Linux distribution is a combination of the Linux kernel, package management system, drivers, scheduler, and other software. They are the operating systems that use Linux as a kernel. 
```

Comment:
Nit: wrapping lines would make this a bit cleaner overall. Not a big deal, but would make it easier to read when reading the markdown directly.